app/models/miq_policy.rb

```diff
@@ -11,12 +11,26 @@ class MiqPolicy < ApplicationRecord
                                  PhysicalServer
                                  Vm).freeze
 
-  POLICY_TOWHAT_APPLIES_TO_CLASSES = TOWHAT_APPLIES_TO_CLASSES.inject({}) { |hsh, key| hsh[key] = ui_lookup(:model => key); hsh }.freeze
-
-  POLICY_MODES = {
-    "control" => _("Control"),
-    "compliance" => _("Compliance")
-  }.freeze
+  def self.POLICY_TOWHAT_APPLIES_TO_CLASSES(opts)
+    policyToWhatAppliesToClasses = TOWHAT_APPLIES_TO_CLASSES.inject({}) { |hsh, key| hsh[key] = ui_lookup(:model => key); hsh }
+    if opts.nil?
+      return policyToWhatAppliesToClasses
+    else
+      return policyToWhatAppliesToClasses[opts]
+    end
```

Comment:
Is the nil returning all a new feature?

Regardless, `return` is redundant here so you can remove it, but we can also one-line this with a ternary for it to read simpler:

```suggestion
    opts.nil? ? policyToWhatAppliesToClasses : policyToWhatAppliesToClasses[opts]
```